### ISSUE TITLE
Error 500 on form results page

### ISSUE THREAD

**Issue by tomekkowalczyk** [mautic/mautic#12044]
### Mautic Version

5.0.x series

### PHP version

8.1

### What happened?

When we enter the form's submitted requests page ex. https://mautic.ddev.site/index_dev.php/s/forms/view/11, I get a 500 error 

![image](https://user-images.githubusercontent.com/39382654/222126147-e0a4b218-dc05-4a08-8c3d-f01025bd0e1b.png)

Trouble is this file: `app/bundles/FormBundle/Views/Field/group.html.twig`

### How can we reproduce this issue?

Step 1:  You enter this page https://mautic.ddev.site/index_dev.php/s/forms/view/{formId}
Step 2:  You should  get a 500 error

### Relevant log output

```shell
{error: {code: 500, text: "Internal Server Error",…}, route: "/index_dev.php/s/forms/edit/11",…}
browserNotifications
{code: 500, text: "Internal Server Error",…}
"Key \"labelAttributes\" for array with keys \"optionlist\" does not exist."
"Internal Server Error"
trace
[{namespace: "", short_class: "", class: "", type: "", function: "",…},…]
flashes
: 
""
newContent
: 
"<!DOCTYPE html>\n<html>\n            <head>\n    
notifications
: 
{content: "↵", lastId: "", hasNewNotifications: false, updateAvailable: true}
route
: 
"/index_dev.php/s/forms/edit/11"
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

**Comment by mabumusa1** [comment]
@tomekkowalczyk thanks for reporting this, I think this is related to the refactoring of the views to twig, if you have the capacity, may you please submit a PR for this issue? I will test and others to merge it soon

**Comment by JoshuaEstes** [comment]
I'm having trouble reproducing @tomekkowalczyk Can you give more information on how the form is setup? What field types, what actions?

One minor note, for the PHP Verison, Mautic currently does not support PHP 8.1 at this time. Using 8.1 I get a lot of errors along with a message saying mautic doesn't support it and to downgrade to 8.0.

**Comment by annamunk** [comment]
@JoshuaEstes I also got this error when trying to save the form with API, PHP 8.0.2:
```
[2023-04-05 08:04:54] mautic.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Key "labelAttributes" for array with keys "optionlist" does not exist." at app\bundles\FormBundle\Resources\views\Field\group.html.twig line 215 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Key \"labelAttributes\" for array with keys \"optionlist\" does not exist. at app\\bundles\\FormBundle\\Resources\\views\\Field\\group.html.twig:215)"} 
```
There seem to be some differences in the structure of the fields, with labelAttributes appearing twice for checkboxes (saving on the Mautic website). In old group template there was also a default empty value if labelAttributes was not set in the properties:
`$optionLabelAttr = (isset($properties['labelAttributes'])) ? $properties['labelAttributes'] : '';`, so maybe that would solve this problem.

Structure of the field that causes an error (saving with API):
```
 [
      "id" => NULL,
      "label" => "Do you want to receive a notification?",
      "showLabel" => true,
      "alias" => "consent",
      "type" => "checkboxgrp", 
      "defaultValue" => NULL,
      "isRequired" => false,
      "validationMessage" => NULL,
      "helpMessage" => NULL,
      "order" => 0,
      "properties"=> [
        "optionlist" => [
          "list" => [
            [
              "label" => "Do you agree?"
              "value" => "1"
            ]
         ]
        ]
      ]
      "labelAttributes" => NULL,
      "inputAttributes" => NULL,
      "containerAttributes" => NULL,
      "leadField" => NULL,
]
```
Structure of the field generated from Mautic website:
```
[
    "label" => "Do you want to receive a notification?",
    "showLabel" => 1,
    "saveResult" => 1,
    "defaultValue" => NULL,
    "helpMessage" => NULL,
    "mappedObject" =>"contact",
    "mappedField" => NULL,
    "isRequired" => 0,
    "validationMessage" =>NULL,
    "properties" => [
      "syncList" => 0,
      "optionlist" => [
        "list" => [
          [
            "label" => "a",
            "value" => "a",
          ],
          [
            "label" => "b",
            "value" => "b",
          ]
        ]
     ]
      "labelAttributes" => NULL,
    ]
    "alias" => "f_name",
    "labelAttributes" => "class="consent"",
    "inputAttributes" => NULL ,
    "containerAttributes" => NULL,
    "parent" => NULL,
    "alwaysDisplay" => 0,
    "isAutoFill" => 0,
    "showWhenValueExists" => 1,
    "showAfterXSubmissions" => NULL,
    "originalMappedField" => NULL,
    "type" => "checkboxgrp",
   ...
 ]
```
![checkbox](https://user-images.githubusercontent.com/102536220/230088645-3ac7180f-5846-44a3-983a-ddcb9fe36725.jpg)

Can you please take a look?

**Comment by LordRembo** [comment]
I think the empty defaults issue has been fixed in a previous merge, as I couldn't reproduce this exact error. But I did get another issue on the exact same use case: the value is not traverable as it is formatted as a string instead of an array. You basically get a string instead of an array if there is only 1 value on the attribute you are adding. Going to fix that and see how deep the rabbit hole goes after that.
